src/odemis/driver/semnidaq.py

```diff
@@ -592,6 +592,19 @@ def __init__(self,
         self.ci_sample_rate = ao_osr / dwell_time  # Hz
         self.ci_samples_n = positions_n * ao_osr
 
+        # These values will be computed later
+        # Chunk: number of samples to read/write at once
+        self.ai_chunk_size = None
+        self.ci_chunk_size = None
+        self.ao_chunk_size = None
+        self.do_chunk_size = None
+
+        # Buffer: number of samples to buffer in memory for the hardware to write/read
+        self.ai_buffer_size = None
+        self.ci_buffer_size = None
+        self.ao_buffer_size = None
+        self.do_buffer_size = None
```

Comment:
All these values are always accessed separately.  We use them like `acq_setting.ai_buffer_size = n` and it would become `acq_setting.io_settings["buffer_size"]["ai"] = n`. So I don't think that will help having a dict.